primitives/runtime/src/traits.rs

```diff
@@ -731,6 +731,15 @@ impl<T> Convert<T, T> for Identity {
 		a
 	}
 }
+
+/// A structure that performs identity conversion of the reference by cloning it.
+pub struct CloneIdentity;
```

Comment:
not needed if these changes accepted - https://github.com/paritytech/polkadot/pull/7461